aws/structure.go

```diff
@@ -1229,6 +1233,9 @@ func flattenESClusterConfig(c *elasticsearch.ElasticsearchClusterConfig) []map[s
 	if c.ZoneAwarenessEnabled != nil {
 		m["zone_awareness_enabled"] = *c.ZoneAwarenessEnabled
 	}
+	if c.ZoneAwarenessConfig.AvailabilityZoneCount != nil {
```

Comment:
Sorry for stupid question, but what happens if `c.ZoneAwarenessConfig` is `nil`?

Reply:
just as if c.ZoneAwarenessEnabled could be nil when an ES domain is not AZ aware, it would also not have an AZ count configured. Technically we could roll this into one if statement and not even set AZ count if zone awareness is disabled. However, it's still entirely possible that the API returns a value for c.ZoneAwarenessEnabled and doesn't return one for AZ count.  Just covering our bases. 👍